src/styles/watch.scss

```diff
@@ -0,0 +1,76 @@
+.stopwatch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  margin: 0 auto;
+  width: 500px;
+  height: 500px;
+  background-color: aquamarine;
+
+  &__face {
+    display: flex;
+    justify-content: center;
+    // size: 80vmin;
+    border: 3px dotted black;
+    width: 400px;
+    height: 400px;
+    border-radius: 50%;
+  }
+
+  &__big-hand {
+    position: absolute;
+    z-index: 1;
+    height: 200px;
+    width: 5px;
+    background-color: blue;
+  }
+
+  &__small-hand {
+    position: absolute;
+    height: 100px;
+    width: 9px;
+    margin: 100px;
+    background-color: red;
+  }
+
+  &__center {
+    position: absolute;
+    align-self: center;
+    z-index: 2;
+    width: 20px;
+    height: 20px;
+    background-color: chocolate;
+    border-radius: 50%;
+  }
+
+  &:hover {
+    .stopwatch__big-hand {
+      animation: sec 60s steps(12) infinite;
```

Comment:
The animation duration for the big hand (seconds hand) is not compliant with the task requirements. It should rotate once every 60 seconds, not every 12 seconds. Replace the animation declaration with: 'animation: sec 60s linear infinite;'.